addons/test_sale_product_configurators/tests/test_sale_product_configurator.py

```diff
@@ -252,6 +252,10 @@ def test_product_attribute_multi_type(self):
                 }),
             ],
         })
+        self.env['res.partner'].create({
+        'name': 'Azure',
+        'email': 'azure@example.com',
+        })
```

Comment:
```suggestion

        self.env['res.partner'].create({
            'name': "Azure",
            'email': "azure@example.com",
        })
```

For strings that are visible to users, we use double quotes

Reply:
hello @mash-odoo  , thanks for letting me know , can I know why is it the case in general or is it just odoo convention? please

Reply:
Hello @omar-sherif9992 
It's actually specific to the `sales` and `e-commerce` modules. The coding guidelines have been recently updated to reflect this.

Reply:
This one is not specific to sales or e-commerce, it's standard python guideline, code should be correctly indented.